portfolio/src/main/webapp/index.html

```diff
@@ -5,14 +5,164 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+
+    <!--Import Google Icon Font-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <!--Import materialize.css-->
+    <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
   </head>
+
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav>
+        <div class="nav-wrapper pink lighten-3">
+        <a href="#" class="brand-logo"><i class="material-icons">star</i>My Portfolio</a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+            <li><a href="#">My Projects</a></li>
```

Comment:
I assume the intention is to make these navigate somewhere eventually, could you add a TODO(helentran): ... comment to make it clear this is intentional.